about.html

```diff
@@ -0,0 +1,83 @@
+     <img id="a-logo" src="./images/nba-logos/nba-logo.svg" alt="Logo">
+     <div id="this-menu">
+     <img id="a-exit" src="./images/close-button.png" alt="Exit Menu" />
+     <ul id="nav-menu">
+        <li id="home"><a class="link" href="index.html">Home</a></li>
+        <li id="menu-1"><a class="link" href="about.html">About</a></li>
+        <li id="menu-2">Program</li>
+        <li id="menu-3">Join</li>
+        <li id="menu-4">Sponsor</li>
+        <li id="menu-5">News</li>
+        <li id="menu-6">Others</li>
+      </ul>
+      </div>
+  </nav>
+  </header>
+    <section id="about">
+    <h1 class="about-title">THIS IS THE NBA FINALS 2023</h1>
+    <p id="about-text">The NBA Finals is the annual Championship series of the National Basketball Association(NBA). The Eastern and Western confernce Champions play a best-of-seven game series to determine the league Champion.The team that wins the series is awarded the Larry O'Brien Championship Trophy.
+    <p class="contact">Please Contact Us per Email for any questions you may have about the NBA<br id="web-link"><a class="ref" href="https//:www.nba.com">NBA.com</a></p>
+    </section>
+    <section id="logo-section">
+        <h3 class="logo-head">NBA LOGO</h3>
+        <p class="logo-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores odit asperiores animi totam obcaecati sunt, neque reiciendis facilis! Temporibus, modi.</p>
+        <img id="logo-img" src="./images/nba-logos/nba-logo.svg" alt="Nba logo">
+    </section>
+    <section id="prev-finals">
+        <h4 class="prev-text">See the past NBA Finals</h4>
+        <p id="prev-sum">Lorem ipsum dolor sit amet consectetur adipisicing elit. Id</p>
+        <div id="finals">
+          <div class="div-1">
+            <img id="first" src="./images/arena/nba-arena.jpeg" alt="2021 NBA Finals">
+            <p class="text-1">2021 NBA Finals</p>
+          </div>
+            <div class="div-2">
+            <img id="second" src="./images/arena/td-garden.png" alt="2020 NBA Finals">
+            <p class="text-2">2020 NBA Finals</p>
+            </div>
+        </div>
+    </section>
+        <section class="sponsors">
+        <h3 id="partneer">Partners</h3>
+        <div id="partners-logo">
+            <img class="partner partner-1" src="./images/sponsors/google.svg" alt="Google">
+            <img class="partner partner-2" src="./images/sponsors/daumkakao.svg" alt="Daumkakao">
+            <img class="partner partner-3" src="./images/sponsors/dell.jpg" alt="Dell">
+            <img class="partner partner-4" src="./images/sponsors/state-farm.svg" alt="State Farm">
+            <img class="partner partner-5" src="./images/sponsors/mozilla.svg" alt="Mozilla">
+            <img class="partner partner-6" src="./images/sponsors/airbnb.svg" alt="Gatorade">
+        </div>
+    </section>
+    <footer id="a-footer">
+        <img class="foot-logo" src="./images/nba-logos/nba-icon.svg" alt="Nba logo">
```

Comment:
- [ ] Please try to remove the space between these two images. 
![image](https://user-images.githubusercontent.com/90222110/183121775-9dec0f7b-7993-4dfd-a15e-a786683832f2.png)

**Design given**
![image](https://user-images.githubusercontent.com/90222110/183121873-341c0ad5-95ef-4a68-a9cb-67e6b440358e.png)

**HINT** You may consider using flex layout and set the `justify-content : space-around`